frontend_tests/test_generator.py

```diff
@@ -16,9 +16,13 @@
     DRIVER = "firefox"
 
 # Test matrix
-SCENARIOS = [scenario_full, scenario_short]
+SCENARIOS = [
+    scenario_full, scenario_short,
+    scenario_no_jupyter, scenario_no_dockerfile,
+    scenario_timeout
+]
 USERS = ["user1", "user2"]
-TIMEOUT = 250
+TIMEOUT = 60
```

Comment:
Do you know why it was set to 250 originally? We don't want to end up getting TIMEOUT errors making travis think it failed. I assumed the 250 was chosen with that in mind.